### DISCUSSION TITLE
No attribute "message_content"

### DISCUSSION THREAD

**Discussion by SansPapyrus683** [Pycord-Development/pycord#1374] [Q&A] [answered]
Hi-
Whenever I try to run the minimal bot example [here](https://docs.pycord.dev/en/master/quickstart.html), I get the following error:
```
Traceback (most recent call last):
  File "C:\Users\kevin\OneDrive\Documents\SNAAAKES\your-mother\main.py", line 6, in <module>
    intents.message_content = True
AttributeError: 'Intents' object has no attribute 'message_content'
```
This is even though I checked this thing here:
<img width="1063" alt="image" src="https://user-images.githubusercontent.com/55369003/169634662-89384c66-7da1-4afd-b8e8-e1092c92c352.png">
However, when I remove that one line, everything seems to run just fine.
Is there something I'm doing wrong?

**Comment by EmmmaTech** [comment, marked as answer]
You might be running Pycord 1.7.3 instead of Pycord 2.0.0. Make sure to install with `pip install py-cord==2.0.0rc.1`

**Reply by SansPapyrus683** [reply]
Thanks for the help!

**Reply by EmmmaTech** [reply]
No problem
